e2e_tests/conftest.py

```diff
@@ -18,53 +16,54 @@ def aau_user_page(
     else:
         raise ValueError(f"Unsupported browser type: {browser_type}")
 
-    browser_instance = browser.launch(headless=True)
+    browser_instance = browser.launch(headless=False)
```

Comment:
I will sort his out in a subsequent PR but for now I wanted it headed to be able to see whats going on